Examples/Tests/linear_compton/analysis_bunch_laser.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
```

Comment:
Could you add a high-level description explaining what the test does?
(e.g. what the simulation scenario is, and what checks we are doing in this file)